django_project/stakeholder/views.py

```diff
@@ -28,10 +28,23 @@
 from core.celery import app
 from frontend.views.base_view import RegisteredOrganisationBaseView
 from frontend.serializers.stakeholder import ReminderSerializer
+from django.contrib.auth.models import User
 logger = logging.getLogger(__name__)
 
 
 
+def check_email_exists(request):
```

Comment:
@tinashechiraya can this API be accessed by public or only by registered user? if this is for registered user, then please protect it by using IsAuthenticated from rest_framework.